modules/custom/activity_send/modules/activity_send_email/activity_send_email.install

```diff
@@ -7,24 +7,6 @@
 
 use Drupal\Core\Database\Database;
 
-/**
```

Comment:
@OH17L, could you tell me why you removed this part?

Reply:
@bramtenhove, I checked message template files and did not see that not all files have E-mail destination. Should I restore this code or will be better if I add this destination to all templates without it?

Reply:
Thanks @OH17L, I think we can leave it out. The `message.template.create_post_profile.yml` file already contains the destination so we should ok.